careers.html

```diff
@@ -1,155 +1,111 @@
-            <hr class="to-animate">
-            <div class="row justify-content-centre">
-                <ul class="to-animate">
-            <section id="fh5co-careers" data-section="careers">
-              <div class="core col-sm-6">
-                <div class="fh5co-post to-animate">
-                  <meta name="viewport" content="width=device-width, initial-scale=1">
-                  <div class="grid-careers">
-                    <div class="circular-careers">
-                      <img src="./images/icons/web-development.png" alt="Full Stack">
-                      <div class="div-text-careers">
-                        <div>
-                          <address>
-                            Full Stack Developers <br> <a
-                              href="mailto:info@dltx.io?subject=Full Stack Developer">Apply</a>
-                          </address>
-                        </div>
-                      </div>
-                    </div>
-                    <div class="circular-careers">
-                      <img src="./images/icons/coding.png" alt="Developers">
-                      <div class="div-text-careers">
-                        <div>
-                          <address>
-                            Developers <br> <a href="mailto:info@dltx.io?subject=Developer">Apply</a>
-                          </address>
-                        </div>
-                      </div>
-                    </div>
-                    <div class="circular-careers">
-                      <img src="./images/icons/development.png" alt="Business development">
-                      <div class="div-text-careers">
-                        <div>
-                          <address>
-                            Business Development <br> <a
-                              href="mailto:info@dltx.io?subject=Business Development">Apply</a>
-                          </address>
-                        </div>
-                      </div>
-                    </div>
-                    <div class="circular-careers">
-                      <img src="./images/icons/agile.png" alt="Agile">
-                      <div class="div-text-careers">
-                        <div>
-                          <address>
-                            Other <br> <a href="mailto:info@dltx.io?subject=Other">Apply</a>
-                          </address>
-                        </div>
-                      </div>
-                    </div>
-                  </div>
-                </div>
-              </div>
-            </section>
-          </div>
-        </div>
+      <div class="role-available to-animate">
+        <img src="images/coding.png" alt="developers"/>
+        <h3>Developers</h3>
+        <p>Our developers typically have some hands-on or personal experience with coding. With the help of our sudo-developers at DLTx, gain more professional hands-on experience building websites and software, and improve your skills.</p>
```

Comment:
Above we dont have a dash...
```suggestion
        <p>Our developers typically have some hands-on or personal experience with coding. With the help of our sudo developers at DLTx, gain more professional hands-on experience building websites and software, and improve your skills.</p>
```